pages/about.html

```diff
@@ -9,34 +9,38 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
-    <title>Intro to Blockchain</title>
+    <title>Blockchain courses</title>
 </head>
 <body>
-    <header>
+    <header class="bg-container-light">
         <section class="bg-dark d-none d-lg-block mb-0">
             <div class="d-flex align-items-center justify-content-end py-2">
-                
+                <a href="#"><img src="/images/social-icons/Linkedinicon.svg" class="w-75"/></a>
+                <a href="#"><img src="/images/social-icons/twitter.svg" class=" w-25 mx-3"/></a>
                 <a class="nav-link link-secondary">English</a>
                 <a class="nav-link link-secondary mx-3">My page</a>
                 <a class="nav-link link-secondary me-5">Logout</a>
             </div> 
         </section>
         <nav class="navbar navbar-expand-lg mt-0">
             <div class="container">
-              
+              <a class="navbar-brand d-none d-lg-block" href="/index.html"><img src="/images/main-logo.png" class="img-fluid main-logo"/>Blockchain Intro</a>
               <button class="navbar-toggler border-0 mb-3" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
                 <span class="navbar-toggler-icon"></span>
               </button>
               
-              <div class="collapse navbar-collapse justify-content-between" id="navbarTogglerDemo02">
-                <button type="button" class="btn-close d-md-none navbar-toggler float-end"
+              <div class="collapse navbar-collapse justify-content-end" id="navbarTogglerDemo02">
+                <button type="button" class="btn-close d-lg-none navbar-toggler float-end"
               data-bs-toggle="collapse" id="menu-close" data-bs-target="#navbarTogglerDemo02"
                aria-controls="navbarTogglerDemo02" aria-expanded="false"
                 aria-label="Close"></button>
-                <a class="navbar-brand" href="/index.html"><img src="/images/main-logo.png" class="img-fluid main-logo"/>Blockchain Intro</a>
+                
                 <ul class="nav flex-column flex-md-row flex-lg-row">
+                  <li class="nav-item d-lg-none">
+                    <a class="nav-link link-secondary active" aria-current="page" href="/index.html">Home</a>
```

Comment:
Same here, kindly update the relative path to the index.html file. Currently, the link to the index file is broken.